auth-service/src/index.ts

```diff
@@ -16,7 +17,16 @@ const port = config.grpcPort;
 const server = new grpc.Server();
 const serviceDiscovery = ServiceDiscovery.getInstance();
 
-server.addService(authHandler.AuthService, new authHandler.AuthHandler());
+const redisClient = redis.createClient(config.redisPort, config.redisHost);
```

Comment:
The client connection could be closed in the shutdown handler for more graceful shutdown

Reply:
That would be smart ... :D I added the the redisclient quit on the `servershutDown()` I don't think the ordering really matters(?), but I added the quit to be before the grpc server shutdown.

Reply:
Actually it may matter: given a situation where the Redis client connection has just been closed, but the server is still accepting gRPC connections, we may end up in an unexpected failure while serving an incoming request 😄This is all theoretical of course, and not likely to happen, but in general I think the server should be shutdown first

Reply:
True, ill change the order